lang/main.json

```diff
@@ -541,8 +541,8 @@
     "profileModal": {
         "displayName": "Display name",
         "email": "Email",
+        "defaultURL": "Server URL",
         "header": "Settings",
-        "serverURL": "Server URL",
```

Comment:
Can't really see, what's the problem with it?

Reply:
all language files would need to be updated. let's leave this change out

Reply:
If we change the key then translation will be broken, because those translation will be for the old key. But it depends if this's been translated already. Given how fresh the profileModal is, I think we can change it.

Reply:
@saghul currently it's only in main.json, but I don't know which one is better so I don't mind if it stays

Reply:
Yeah, see my other comment though.

Reply:
Yes, it's the server URL. "default URL" feels more like a whole thing including room name or whatever.